auslib/test/web/test_client.py

```diff
@@ -642,6 +690,30 @@ def testGetWithResponseProductsWithOneRule(self):
 """)
         self.assertEqual(returned.toxml(), expected.toxml())
 
+    def testSystemAddonsBlobWithUninstall(self):
+        ret = self.client.get('/update/4/systemaddons-uninstall/1.0/1/z/p/a/b/c/d/1/update.xml')
+        self.assertEqual(ret.status_code, 200)
+        self.assertEqual(ret.mimetype, 'text/xml')
+        returned = minidom.parseString(ret.data)
+        expected = minidom.parseString("""<?xml version="1.0"?>
+<updates>
+    <addons>
+    </addons>
+</updates>
+""")
+        self.assertEqual(returned.toxml(), expected.toxml())
+
+    def testSystemAddonsBlobWithoutUninstall(self):
+        ret = self.client.get('/update/4/systemaddons/1.0/1/z/p/a/b/c/d/1/update.xml')
+        self.assertEqual(ret.status_code, 200)
+        self.assertEqual(ret.mimetype, 'text/xml')
+        returned = minidom.parseString(ret.data)
+        expected = minidom.parseString("""<?xml version="1.0"?>
+<updates>
+</updates>
+""")
+        self.assertEqual(returned.toxml(), expected.toxml())
+
```

Comment:
These tests are a good start, but please also add some tests in auslib/test/blobs so we can test them directly.